packages/amplify-cli-core/API.md

```diff
@@ -981,6 +1004,12 @@ export class JSONUtilities {
     } | undefined) => void;
 }
 
+// Warning: (ae-forgotten-export) The symbol "PackageLock" needs to be exported by the entry point index.d.ts
+// Warning: (ae-forgotten-export) The symbol "YarnLock" needs to be exported by the entry point index.d.ts
+//
+// @public (undocumented)
+export type Lockfile = PackageLock | YarnLock;
+
```

Comment:
1. see warnings, they say that PackageLock and YarnLock should also be exported.
2. do we need to export this at all? if we hide `AmplifyNodePkgDetector.parseLockFile` then we could hide this type as well.